src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -261,7 +261,7 @@ void BaseCouplingScheme::extrapolateData(DataMap &data)
       utils::shiftSetFirst(pair.second->oldValues, values);
     }
   } else {
-    PRECICE_ERROR("Called extrapolation with order != 1,2!");
+    PRECICE_ASSERT(false, "Extrapolation order is invalid.");
```

Comment:
Why `PRECICE_ASSERT(false, ...)` and not just `PRECICE_ERROR(...)`?

Reply:
This case should have been caught in the configuration.
It is thus an precice-internal problem and  should trigger an assertions.